docs/integrations.md

```diff
@@ -0,0 +1,138 @@
+---
+layout: default
+title: Integrations
+has_children: false
+has_toc: false
+nav_order: 9
+---
+
+# Integration with SmartClean Matrix
+_Integrations allow you to set up access to data and events from our platform in a standard way._
+
+## Integration mechanisms supported
+1. Receive data and events from our platform on your web service.
+2. Query certain information from our platform by making web requests.
+3. Query data and events from an SQL database.
+4. Query data from our platform by using our SDK for certain programming languages.
+
+[comment]: <> (Should create a sub page for each of these options ? )
```

Comment:
We can point to specific pages currently if they exist. For example the links to SDK, link to dedicated db.

Reply:
Resolved by commit:
https://github.com/smartclean/smartclean.github.io/commit/46a827ac0238774caae60421357e894dec515a7d